src/api/FactsController.js

```diff
@@ -0,0 +1,14 @@
+export async function retrieveAnimalFact() {
+  const animalTypes = ['dog', 'cat'];
+  const randomAnimalType = randomIntFromInterval(0, animalTypes.length - 1);
+  const animalType = animalTypes[randomAnimalType];
+  const FACT_URL = `https://cat-fact.herokuapp.com/facts/random?animal_type=${animalType}&amount=1`;
```

Comment:
in this case, the variable should be in the camel case (factUrl). We keep the CAPITALIZED_WITH_UNDERSCORES convention to constants outside function block